metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -365,10 +396,8 @@ def get_tables_for_dataset(
                 ddl=table.ddl,
                 expires=tables[table.table_name].expires if tables else None,
                 labels=tables[table.table_name].labels if tables else None,
-                time_partitioning=PartitionInfo.from_time_partitioning(
-                    tables[table.table_name].time_partitioning
-                )
-                if tables and tables[table.table_name].time_partitioning
+                partition_info=PartitionInfo.from_table_info(tables[table.table_name])
```

Comment:
So we previously assumed all Partitioned Tables were time partitioned? And now we know there's another default case of range partitioning that we are handling?

Reply:
got it - makes sense